include/fuzzuf/algorithms/aflplusplus/aflplusplus_util.hpp

```diff
@@ -45,9 +49,9 @@ double ComputeWeight(const State &state,
     weight *= std::log10(hits) + 1;
   }
 
-  weight *= (avg_exec_us / testcase.exec_us);
-  weight *= (std::log(testcase.bitmap_size) / avg_bitmap_size);
-  weight *= (1 + (testcase.tc_ref / avg_top_size));
+  weight *= ((avg_exec_us + epsilon) / (testcase.exec_us + epsilon));
+  weight *= (std::log(testcase.bitmap_size + 1) / avg_bitmap_size);
```

Comment:
While this case is unlikely to occur, for consistency, would it be possible to insert an epsilon here as well?
```suggestion
  weight *= (std::log(testcase.bitmap_size + 1) / (avg_bitmap_size + epsilon));
```

Reply:
oops you are right.
Fixed in https://github.com/fuzzuf/fuzzuf/pull/97/commits/862c8fe4c05fc7fe0bada718b30e115397fe5a3a.
Also I realized that, among the three multiplying terms we are considering, the last two terms have a kind of "integrality" (`bitmap_size` and `tc_ref` are integer variables), so for example `avg_bitmap_size` becomes 0 iff all `bitmap_size` are 0. Therefore, we don't need to add `epsilon` to the numerators. 
On the otherhand, both the denominator and numerator of the first term are double variables, both should be added `epsilon` to.